### PR TITLE
Update 02_basics.adoc

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: documentation/modules/ROOT/pages/02_basics.adoc [hunk 1]
```diff
@@ -101,15 +101,15 @@ Quarkus CLI::
 quarkus test
 ----
 
-NOTE: `quarkus test` starts tests in continuous testing mode (more about this below) so to return to the command line after the tests ran you will need to press (q)
+NOTE: `quarkus test` starts tests in continuous testing mode (more about this below) so to return to the command line after the tests ran you will need to press kbd:[q].
```
Comment (gsmet) [inline documentation/modules/ROOT/pages/02_basics.adoc:104]: This needs to be checked in your environment but it should be supported by Asciidoc: https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/

File: documentation/modules/ROOT/pages/02_basics.adoc [hunk 2]
```diff
@@ -121,13 +121,13 @@ NOTE: If you use `tutorial-tools`, your host `quarkus-tutorial/work` contains th
...
 
-Stop the Quarkus application that might be running in `jvm` or `native` mode.
+Stop the Quarkus application that might be running in JVM or native mode.
 
 _Live Coding_ or _Development mode_ enables hot deployment with background compilation, which means that when you modify your Java files and/or your resource files and refresh your browser, these changes will automatically take effect.
 
 This is probably one of the best features of Quarkus: enabling a very fast and productive iterative feedback loop.
 
-Let's start the _Live Coding_ mode by invoking `mvn`. You probably won't need to stop/start Quarkus again during this tutorial:
+Let's start the _Live Coding_ mode by using the `dev` command. You probably won't need to stop/start Quarkus again during this tutorial:
```
Comment (gsmet) [inline documentation/modules/ROOT/pages/02_basics.adoc:130]: You're suggesting either the CLI or Maven so let's make sure we are a bit more generic.

File: documentation/modules/ROOT/pages/02_basics.adoc [hunk 3]
```diff
@@ -182,6 +182,6 @@ Press `r` and the tests will start running. You should see the status change dow
 All 1 test is passing (0 skipped), 1 test was run in 11705ms. Tests completed at 16:02:32.
 ----
 
-TIP: If you don’t want to have continuous testing enabled, you can change this by adding `quarkus.test.continuous-testing=disabled` in `application.properties`. 
+TIP: If you don’t want to have continuous testing enabled, you can change this by adding `quarkus.test.continuous-testing=disabled` to your `src/main/resources/application.properties`.
```
Comment (gsmet) [inline documentation/modules/ROOT/pages/02_basics.adoc:185]: I don't think you presented the `applications.properties` yet so I think it's probably better to have the full path.

File: documentation/modules/ROOT/pages/02_basics.adoc (hunk 1 above)
Comment (kdubois) [reply documentation/modules/ROOT/pages/02_basics.adoc:104]: let's try it :D

**Pull request merged**